app/views/projects/_form.html.erb

```diff
@@ -293,6 +293,7 @@ databases when reporting vulnerabilities.
           </li>
 
    <%# Note: render() accepts met_suppress: ..., unmet_placeholder:  ... %>
+      <% cache [project, is_disabled, @result] do %>
```

Comment:
@dankohn - I already knew that, but thanks!  I've also moved the cache to cover more area... and in the process, I now see why it didn't work before.  There's another <% end %> that caused our previous version to fail.

With these tweaks I'm getting an 80% increase in speed in the server-side completion time when displaying project entries for read.